039-AKSForEnterprise/Coach/03-aks_monitoring.md

```diff
@@ -0,0 +1,168 @@
+# 3. Monitoring - Coach's Guide

```

Comment:
Change title to: "Challenge 3: Monitoring - Coach's Guide"

Reply:
Done!